ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/logs/NewReplayModal.tsx

```diff
@@ -103,7 +101,7 @@ export default function NewReplayModal({
                 Specify a time range to replay function runs from.
               </p>
             </div>
-            <DateTimeRangeInput onChange={setTimeRange} />
+            <TimeRangeInput onChange={setTimeRange} />
```

Comment:
**question (non-blocking)**: why did you rename it?

Reply:
Seemed 0.1% less verbose, since the only kind of time input is a datetime